demos/storybook/package.json

```diff
@@ -25,6 +25,7 @@
         "@angular/material": "^11.2.8",
         "@angular/platform-browser": "~11.2.9",
         "@angular/platform-browser-dynamic": "~11.2.9",
+        "@pxblue/angular-components": "5.0.1",
```

Comment:
does demo storybook package.json need update here

Reply:
Storybook won't be updated since it won't get deployed anywhere.